Source/PeleLM.cpp

```diff
@@ -5381,10 +5378,15 @@ PeleLM::advance (Real time,
    if (do_mom_diff == 0) {
       velocity_advection(dt);
    }
+   VisMF::Write(get_new_data(State_Type),"newVelAfterAdv");
+   VisMF::Write(get_old_data(State_Type),"oldVelAfterAdv");
+   VisMF::Write(get_rho_half_time(),"rhoHalfAfterAdv");
+   VisMF::Write(*aofs,"aofsAfterAdv");
    
```

Comment:
Remove these, or put them behind an if so that we can shut them off.

Reply:
They've been removed (marked as outdated here).